L/libjulia/common.jl

```diff
@@ -0,0 +1,161 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg.Types
+
+# Collection of sources required to build GMP
+function configure(version)
+    name = "libjulia"
+
+    checksums = Dict(
+        v"1.4.2" => "76a94e06e68fb99822e0876a37c2ed3873e9061e895ab826fd8c9fc7e2f52795",
+        v"1.5.1" => "1f138205772eb1e565f1d7ccd6f237be8a4d18713a3466e3b8d3a6aad6483fd9",
```

Comment:
Can we (eventually!) get 1.3.1, too? Or is there a major stumbling block there?

Of course only after 1.5 works, and only if it can be done with reasonable effort.